input/vulnerabilities/Mtkbf.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name" : "Mtkbf exploit",
+  "CVE" : [],
+  "Coordinated_disclosure" : "true",
+  "Categories" : ["system"],
+  "Details" : [["Memory write vulnerabilities allow a local user to gain privileges", "mtkbf-ele7enxxh-blog"]],
+  "Discovered_by" : [["KeenTeam", "mtkbf-ele7enxxh-blog"]],
+  "Discovered_on" : [["2015"]],
+  "Submission" : [{"by":"dcc52", "on":"2019-07-08"}],
+  "Reported_on" : [["2015"]],
```

Comment:
Can we get a more precise date?

Reply:
It's "early 2015" if there's a way of putting that in. Trying to find anything more specific is just pointing me towards a conference paper in Chinese which anyway seems to be rather difficult to track down

Reply:
Run with the end of April?

Reply:
Done